packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -437,16 +437,25 @@ export const workflowsRouter = router({
                   }),
                   organizer: booking.user
                     ? {
-                        language: { locale: booking.user.locale || "" },
-                        name: booking.user.name || "",
-                        email: booking.user.email,
-                        timeZone: booking.user.timeZone,
+                        language: { locale: booking?.user[0]?.locale || "" },
+                        name: booking?.user[0]?.name || "",
+                        email: booking?.user[0].email,
+                        timeZone: booking?.user[0].timeZone,
                       }
                     : { name: "", email: "", timeZone: "", language: { locale: "" } },
                   startTime: booking.startTime.toISOString(),
                   endTime: booking.endTime.toISOString(),
                   title: booking.title,
-                  language: { locale: booking?.user?.locale || "" },
+                  language: { locale: ctx.user?.locale || "" },
+                  ...(booking.user.length > 1 && {
+                    team: {
+                      members: booking.user.reduce((teamArray, user, index) => {
+                        if (index !== 0)
+                          teamArray.push({ id: user.id, name: user.name || "", email: user.email });
+                        return teamArray;
+                      }, [] as { id: number; name: string; email: string }[]),
+                    },
+                  }),
                 };
                 if (
                   step.action === WorkflowActions.EMAIL_HOST ||
```

Comment:
@CarinaWolli, for workflows, should `EMAIL_HOST` be just the organizer or all team members? Another idea I had is whether we should add `TEAM_MEMBERS` as a separate option.

Reply:
I prefer sending it only to the organizer for now and introducing a new action to send it to all team members (like you said to add TEAM_MEMBERS)